core/runtime/binaryen/runtime_api/runtime_api.hpp

```diff
@@ -65,9 +96,23 @@ namespace kagome::runtime::binaryen {
     outcome::result<R> execute(std::string_view name,
                                CallPersistency persistency,
                                Args &&... args) {
+      return executeMaybeAt<R>(
+          name, boost::none, persistency, std::forward<Args>(args)...);
+    }
+
+   private:
+    template <typename R, typename... Args>
+    outcome::result<R> executeMaybeAt(
```

Comment:
Interesting name) Maybe makes sense to explain in comments what function is doing